Dockerfile

```diff
@@ -3,11 +3,8 @@ MAINTAINER kaakaa <stooner.hoe@gmail.com>
 
 RUN apk update
 
-RUN apk add wget unzip fontconfig libreoffice
-RUN wget -O ipa-font.zip 'https://osdn.jp/frs/redir.php?m=iij&f=%2Fipafonts%2F51868%2FIPAfont00303.zip'
```

Comment:
Unfortunately, if deleting instruction of installing `ipa-font`, slide with japanese font can't be convert properly.
Would you please revert these instruction? (It's good for me to introduce `msttcorefonts-installer`)